zipline/gens/tradesimulation.py

```diff
@@ -266,11 +268,14 @@ def _call_handle_data(self):
         Call the user's handle_data, returning any orders placed by the algo
         during the call.
         """
+        time_start = utcnow()
         self.algo.event_manager.handle_data(
             self.algo,
             self.current_data,
             self.simulation_dt,
         )
+        time_end = utcnow()
+        self.algo.call_handle_data_duration.append((time_end-time_start).total_seconds()*1000)
```

Comment:
If I use time.time(), would I be able to keep the time calculations in zipline instead of needing to move it to the internal wrapper of handle_data (which I assume is call_algorithm_handle_data in algoproxy)?